demos/universal-sentence-encoder/README.md

```diff
@@ -50,17 +51,18 @@ The layer SentencepieceTokenizer expects on the input a list of strings. The CPU
 
 Until it is published, the docker image with OpenVINO Model Server including the CPU extension has to be built using the commands:
 
-```
+```bash
 git clone -b develop https://github.com/openvinotoolkit/model_server
```

Comment:
I'm not sure about using "-b develop" here. Is it needed if this will be merged into develop branch?